gst_plugins/python/savant_rs_video_demux.py

```diff
@@ -114,27 +115,40 @@
         False,
         GObject.ParamFlags.READWRITE,
     ),
+    'ingress-module': (
+        str,
+        'Python module',
+        'Python module name to import or module path.',
```

Comment:
Describe what is this python module. And for other properties too.

Reply:
Improved the description.